libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/TripServiceImpl.kt

```diff
@@ -0,0 +1,145 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+import com.mapbox.android.core.location.LocationEngine
+import com.mapbox.android.core.location.LocationEngineCallback
+import com.mapbox.android.core.location.LocationEngineRequest
+import com.mapbox.android.core.location.LocationEngineResult
+import com.mapbox.navigation.base.trip.NavigationTripDescriptor
+import com.mapbox.navigation.base.trip.TripNotification
+import com.mapbox.navigation.base.trip.TripService
+import com.mapbox.navigation.navigator.MapboxNativeNavigatorImpl
+import com.mapbox.navigation.utils.extensions.ifNonNull
+import java.util.concurrent.atomic.AtomicBoolean
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.channels.ConflatedBroadcastChannel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+import timber.log.Timber
+
+const val NAVIGATION_SERVICE_ID = 0xff00aa
+const val UPDATE_INTERVAL_IN_MILLISECONDS: Long = 1000
+const val FASTEST_UPDATE_INTERVAL_IN_MILLISECONDS: Long = 500
+
+@ExperimentalCoroutinesApi
+internal object NavigationNotificationService : Service() {
```

Comment:
Same as above re: shouldn't this object live under `libtrip-service` module?

Reply:
file removed